README.md

```diff
@@ -48,34 +48,33 @@ Each MD page in `/faq/` has one or more taxonomy defined in header. E.g.:
 
 ```yaml
 taxonomies:
-  faq: ["Android"]
+  faq: ["app"]
 ```
 
 Zola collects all such taxonomies:
 
-| File                 | Taxonomy key | Taxonomy value |
-| -------------------- | ------------ | -------------- |
-| android-lags.md      | `faq`        | `Android`      |
-| android-logs.md      | `faq`        | `Android`      |
-| general-team.md      | `faq`        | `General`      |
-| general-bugreport.md | `faq`        | `General`      |
-| ios-versions.md      | `faq`        | `iOS`          |
-| map-longtap.md       | `faq`        | `Map`          |
-| ...                  | ...          | ...            |
+| File                                    | Taxonomy key | Taxonomy value |
+|-----------------------------------------| ------------ |----------------|
+| map/can-find-position/index.md          | `faq`        | `map`          |
```

Comment:
```suggestion
| map/can-not-find-position/index.md      | `faq`        | `map`          |
```

Reply:
Should it be capitalized? e.g. Map, App?

Reply:
Renamed `faq/map/can-find-position` to `faq/map/can-not-find-position` 
Updated "Taxonomy value" column